opendcs-rest-api/src/test/java/org/opendcs/odcsapi/res/it/AppResourcesIT.java

```diff
@@ -0,0 +1,496 @@
+package org.opendcs.odcsapi.res.it;
+
+import java.sql.Date;
+import java.time.Instant;
+import java.util.Properties;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.MediaType;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.filter.log.LogDetail;
+import io.restassured.filter.session.SessionFilter;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.opendcs.odcsapi.beans.ApiLoadingApp;
+import org.opendcs.odcsapi.fixtures.DatabaseContextProvider;
+import org.opendcs.odcsapi.fixtures.ResourcesTestBase;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+
+@Tag("integration")
+@ExtendWith(DatabaseContextProvider.class)
+final class AppResourcesIT extends ResourcesTestBase
+{
+	private static Long appid;
+	private static SessionFilter sessionFilter;
+	private static Properties properties;
+	private static final ObjectMapper objectMapper = new ObjectMapper();
+
+	@BeforeAll
+	static void setUpAll()
+	{
+		properties = new Properties();
+		properties.setProperty("startCmd", "$DCSTOOL_HOME\\bin\\compproc.bat");
+	}
+
+	@BeforeEach
+	void setUp() throws Exception
+	{
+		setUpCreds();
+		appid = null;
+		sessionFilter = new SessionFilter();
+
+		given()
```

Comment:
use the same indenting strategy we did for CDA

Reply:
Updated formatting; looks like it was lost when moving to the new branch